dogstatsd/src/datadog.rs

```diff
@@ -20,8 +21,13 @@ pub struct DdApi {
 
 impl DdApi {
     #[must_use]
-    pub fn new(api_key: String, site: String, https_proxy: Option<String>) -> Self {
-        let client = match Self::build_client(https_proxy) {
+    pub fn new(
```

Comment:
nitpick: I think it's ok right now, but if we go beyond four parameters here we should consider implementing a Dogstatsd builder.

Reply:
Agree, I'll do it in another PR tho!